public/sass/elements/forms/_form-validation.scss

```diff
@@ -10,7 +10,8 @@
 
   // Form inputs should have a red border
   // Add a red border to .form-controls that are direct descendants
-  > .form-control {
+  > .form-control,
+  > fieldset > .form-date > .form-group > .form-control {
```

Comment:
Can we change the new selector to just `.form-date .form-control`? Walking the tree doesn’t take any time in modern browsers, and it reduces the specificity hit.